src/main/java/svc/profiles/LocalWithHSQLAccessProfile.java

```diff
@@ -16,13 +16,16 @@
 @Configuration
 @Profile("localWithHSQLAccess")
 public class LocalWithHSQLAccessProfile {
-	@Bean
+/*	@Bean
```

Comment:
Is the commented-out code valuable in any way?

Reply:
No, it was commented out while testing the data-local.sql way of importing test data.